modules/standard/Math.chpl

```diff
@@ -1057,6 +1057,72 @@ module Math {
     return truncf(x);
   }
 
+
+  /* Returns the Bessel function of the first kind of order `0` of `x`. */
+  extern "j0f" proc j0(x: real(32)): real(32);
+
+  /* Returns the Bessel function of the first kind of order `0` of `x`. */
+  inline proc j0(x : real(64)): real(64) {
+    extern proc j0(x: real(64)): real(64);
+    return j0(x);
+  }
+
+
+  /* Returns the Bessel function of the first kind of order `1` of `x`. */
+  extern "j1f" proc j1(x: real(32)): real(32);
+
+  /* Returns the Bessel function of the first kind of order `1` of `x`. */
+  inline proc j1(x : real(64)): real(64) {
+    extern proc j1(x: real(64)): real(64);
+    return j1(x);
+  }
+
+
+  /* Returns the Bessel function of the first kind of order `n` of `x`. */
+  extern "jnf" proc jn(n: int, x: real(32)): real(32);
```

Comment:
Note that `int != c_int`. Chapel `int` is implicitly `int(64)`, whereas `c_int` is a type alias for `int(32)`, the default `int` size for C.

Reply:
The default int size for C isn't even necessarily int(32) although that is the currently common configuration.

Reply:
@nimitbhardwaj  - this still needs to be updated to `n: c_int`

Reply:
This should look more like jn below, since we have to cast. The extern proc must have c_int not int in the type.